Dockerfile-alpine.template

```
@@ -116,6 +116,10 @@ RUN set -eux; \
 		echo "$adapter:" >> ./config/database.yml; \
 		echo "  adapter: $adapter" >> ./config/database.yml; \
 	done; \
+{{ if ([ "5.0" ] | index(env.version)) and ([ "alpine3.21" ] | index(env.variant)) then ( -}}
```

Oh, this will not apply on Alpine 3.22 as-is, which is likely to have the same issue, so I think we should flip the check:

```suggestion
{{ if env.version == "5.0" and (env.variant | startswith("alpine")) and env.variant != "alpine3.20" then ( -}}
```